model/infinispan/src/main/java/org/keycloak/models/sessions/infinispan/remotestore/RemoteCacheSessionsLoader.java

```diff
@@ -65,42 +57,10 @@ public void init(KeycloakSession session) {
 
     @Override
     public RemoteCacheSessionsLoaderContext computeLoaderContext(KeycloakSession session) {
-        RemoteCache remoteCache = getRemoteCache(session);
-        int sessionsTotal = remoteCache.size();
-        int ispnSegments = getIspnSegmentsCount(remoteCache);
-
-        return new RemoteCacheSessionsLoaderContext(ispnSegments, sessionsPerSegment, sessionsTotal);
+        return new RemoteCacheSessionsLoaderContext(1, sessionsPerSegment);
```

Comment:
Does the constructor has to be parametrized when we hardcore "1" here and other usages are just from tests (Those tests should be changed, if makes sense, or removed).

Reply:
Thanks, cleaned up.